.circleci/config.yml

```diff
@@ -2,17 +2,19 @@ version: 2.1
 orbs:
   codecov: codecov/codecov@1.0.2
 jobs:
+
   build:
     docker:
-      - image: circleci/openjdk:8-jdk-browsers
+      - image: circleci/openjdk:8-jdk
```

Comment:
Is this an intentional change?

Reply:
From referencing CircleCI docs it tended to refer to 8-jdk over 8-jdk-browsers. I figured we weren't using browsers in our tests. Happy to revert.

Reply:
As long as there is a reason for the change then that's fine